src/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -318,6 +321,7 @@ private function addContentTypeMetaTag(string $html): string
         } else {
             $reworkedHtml = self::CONTENT_TYPE_META_TAG . $html;
         }
+        \assert($reworkedHtml !== '');
```

Comment:
If `@param non-empty-string $html` is added, this may not be needed.

Reply:
This still is necessary as PHPStan currently does not know how our preg class works.

Reply:
I think it might struggle even if we used `preg_replace()` directly.